va/va_trace.c

```diff
@@ -855,6 +884,10 @@ void va_TraceEnd(VADisplay dpy)
     if (!pva_trace)
         return;
 
+#if defined(_WIN32)
+    DeleteCriticalSection(&pva_trace->resource_mutex);
+    DeleteCriticalSection(&pva_trace->context_mutex);
+#endif
```

Comment:
looks no pthread_mutex_destroy called before?

Reply:
You're right. I'll add it now so there are no leaks on any of the platforms